docs/bcd-sync.md

```diff
@@ -0,0 +1,46 @@
+
+## BCD keys in `compat_features`
+
+The source YAML files, that define each feature in the web-features, might contain a `compat_features` field.
+
+When present, the `compat_features` field consists of a list of browser-compat-data (BCD) entry keys that make up this feature (e.g., `css.properties.background-color`).
+
+If the `compat_features` field is not present in a feature's source YAML file, the resulting `.yml.dist` file (generated by the `dist` script) will contain a `compat_features` field that contains the BCD entry keys that were tagged with `web-features:<feature-id>` in the BCD repository, if any.
+
+The `compat_features` field in a feature's source YAML file always takes precedence over tags defined in BCD.
+
+## Tagging keys in BCD
+
+One way to map keys to web-features is that keys are tagged in BCD from inception. This is more likely for small, incremental changes to existing features.
+
+For example, a typical scenario could look like this:
+
+- a new key appears in BCD (e.g., via a [Collector](https://github.com/openwebdocs/mdn-bcd-collector) PR).
+- The author of the BCD PR knew which feature the key belongs to (e.g., due to a key being renamed) and tags the key with the correct `web-features:*` tag.
+- A few days later, that data is released by BCD. That triggers the next `@mdn/browser-compat-data` Dependabot upgrade PR, where we regenerate web-features data based on the updated tag and remove the `compat_features` list from the YAML file (if the list and tagged keys match).
+
+ If necessary (e.g., a tag was assigned by BCD maintainers in error), a web-features maintainer can update a `compat_features` list in web-features and the data gets synchronized to BCD as described in the next section.
+
+## Listing keys in web-features source YAML files
+
+The other way to map keys to web-features is to list them in the source YAML files. In this scenario, web-features receives unaccounted-for keys in the `features/draft` folder which will have to be looked into by web-features maintainers and will later be added to BCD as tags automatically.
+
+For example, a typical scenario would look like this:
+
+- a new key appears in BCD (e.g., via a [Collector](https://github.com/openwebdocs/mdn-bcd-collector PR).
+- The author of the BCD PR doesn't know or doesn't care which feature that key belongs to and does not tag it.
+- A few days later, that data is released by BCD. That triggers the next `@mdn/browser-compat-data` Dependabot upgrade PR. After that PR merges, we run the "Update draft features" workflow, which makes the new keys to appear in the `features/draft` folder.
+- At this point, a web-features maintainer can assign the key to an existing feature or author a new feature.
+- A few days later, that data is released by web-features. When BCD upgrades `web-features`, it'll automatically apply tags to each feature according to the `compat_features` list.
```

Comment:
Tags get applied to *keys* in BCD, right?

Also, does "apply tags" mean: "add" or "add and remove" (i.e. "synchronize")?

```suggestion
- A few days later, that data is released by web-features. When BCD upgrades `web-features`, it'll automatically adds/removes tags to/from each key according to the `compat_features` list.
```